server/spinwick.go

```diff
@@ -290,12 +290,14 @@ func (s *Server) createSpinWick(pr *model.PullRequest, size string, withLicense
 	// }
 
 	installationRequest := &cloudModel.CreateInstallationRequest{
-		OwnerID:  ownerID,
-		Version:  version,
-		Image:    image,
-		DNS:      fmt.Sprintf("%s.%s", ownerID, s.Config.DNSNameTestServer),
-		Size:     size,
-		Affinity: "multitenant",
+		OwnerID:   ownerID,
+		Version:   version,
+		Image:     image,
+		DNS:       fmt.Sprintf("%s.%s", ownerID, s.Config.DNSNameTestServer),
+		Size:      size,
+		Affinity:  "multitenant",
+		Database:  "aws-multitenant-rds",
```

Comment:
The postgres multitenant value is actually different: https://github.com/mattermost/mattermost-cloud/blob/a981c07c93e5a49d26710623eaa9eb0b788bbd37/model/installation_database.go#L30

I would make sure the mattermost-cloud repo dependancy is up to date and use `cloudModel.InstallationDatabaseMultiTenantRDSPostgres` instead.

Reply:
right! thanks for the catch!

Reply:
done @gabrieljackson